hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                        </thead>
+                        <tbody>
+                        {% if deleted_resources %}
+                            {% for res in deleted_resources %}
+                                <tr>
+                                    <td>
+                                        {{ res.resource_id }}
+                                    </td>
+                                    <td>
+                                        {{ res.resource_title }}
+                                    </td>
+                                    <td>
+                                        {{ res.resource_type }}
+                                    </td>
+                                    <td>
+                                        {% if res.deleted_by.first_name %}
+                                            <strong><a href='/user/{{ res.deleted_by.pk }}/' target="_blank">{{res.deleted_by.first_name}} {{res.deleted_by.last_name }}</a></strong>
+                                        {% else %}
+                                            <strong><a href='/user/{{ res.deleted_by.pk }}/' target="_blank">{{ res.deleted_by.username }}</a></strong>
+                                        {% endif %}
+                                    </td>
+                                    <td>
+                                        {{ res.date_deleted }}
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                         {% endif %}
+                        </tbody>
+                    </table>
+                    <div id="clear-deleted-res-warning" style="display: None">It may take seconds to clear deleted resources and update bag. Please keep this window open and wait...</div>
+                </div>
+                <div class="modal-footer">
+                    {% if edit_mode|lower == "true" %}
+                    <button type="button" class="btn btn-success" id="clear-deleted-res-btn" onclick="clear_deleted_resources_table('{{collection_res_id}}');">
+                        Clear and Update Bag
+                    </button>
+                    {% endif %}
+                    <button type="button" class="btn btn-default" data-dismiss="modal" id="save-collection-btn-cancel">
+                       Close
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    <script type="text/javascript" src='{% static "js/jquery.dataTables.js" %}'></script>
+
+    {# put resource specific js below #}
+    <script type="text/javascript">
+
+        var paging = false;
+        var page_length = 15;
+        var page_type = "numbers";
+        var page_length_change = false;
+        $(document).ready(function()
+        {
+            // disable "Publish" btn
+            $("#publish").hide();
+            $("#new-version").hide();
```

Comment:
@zhiyuli I guess initially we don't want to expose versioning and publishing functionalities to collection resource until we iron out details later and may expose it later in the second phase.